common/src/main/java/us/myles/ViaVersion/protocols/protocol1_14to1_13_2/MetadataRewriter.java

```diff
@@ -39,6 +39,12 @@ public static void handleMetadata(int entityId, Entity1_14Types.EntityType type,
                 if (metadata.getId() > 5) {
                     metadata.setId(metadata.getId() + 1);
                 }
+                if( metadata.getId() == 8 && type.isOrHasParent(Entity1_14Types.EntityType.LIVINGENTITY)) {
```

Comment:
Add a whitespace after the two `if`s and remove the one before metadata, but else the pr is good to go :p